homework.md

```diff
@@ -3,111 +3,109 @@
 ## Question 1
 
 What is 2 + 2?
+Answer 1: 4
 
-5
 
 ## Question 2
 
 What is the opposite of Black?
-
-Purple
+Answer 2: white
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
-
-1322
+Answer 3: 12240
 
 ## Question 4 
 
 What does HTML stand for?
-
-Hyper Tall Massive Language
+Answer 4: Hyper Text Markup Language
 
 ## Question 5
 
 What is CSS useful for?
+Answer 5: It make easier to separate the style from the structure, and you can work on its own.
 
-Making a website go faster
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
-
+Answer 6:
 ```css
-#h1 {
-
+ h1 {
 }
 ```
 
 ## Question 7
 
 What is JavaScript?
-
-A programming language used on the internet
+Answer 7: JavaScript is a programming language that is used for web developments.
```

Comment:
Also worth mentioning that it's used to add special effects and dynamic behaviour to the webpage :)